### PR TITLE
Enhance error message

### LINKED ISSUE #873: Enhance error message
`Error running: loop_runs_on_this_container: Error binding step input: 'container_id' should have a String value.`

actual value, actual type?

### PR DESCRIPTION
**Pull request by cid77**
Fixes #873 

Signed-off-by: Ciprian Duma ciprian-eugen.duma@hpe.com

### CODE REVIEW

**Comment by gituser173** [comment]
👍 looks good

File: cloudslang-commons/src/main/java/io/cloudslang/utils/ValidationUtils.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *******************************************************************************/
+package io.cloudslang.utils;
+
+import java.io.Serializable;
+
+public class ValidationUtils {
+
+    public static void validateStringValue(String errorMessagePrefix, Serializable value) {
```
Comment (lucian-cm) [inline cloudslang-commons/src/main/java/io/cloudslang/utils/ValidationUtils.java]: this name is a little confusing since it validates a `Serializable`

**Comment by lucian-cm** [comment]
Looks good 👍

**Comment by lucian-cm** [comment]
Merging as comments from everyone were fixed.

**Pull request merged**